tests/materials/processing/strategies/test_aggregates_strategy.py

```diff
@@ -19,8 +42,32 @@ def test_gather_composition_properties_adds_all_properties():
         composition=composition,
     )
 
-    result = strategy.gather_composition_information(aggregates)
+    result = AggregatesStrategy.gather_composition_information(aggregates)
     assert result == ['Fine Aggregates: 123.45, ',
                       'Coarse Aggregates: 67.89, ',
                       'FA Density: test FA density, ',
                       'CA Density: test CA density, ']
+
+
+def test_convert_to_multidict_adds_all_properties():
+    composition = Composition(
+        fine_aggregates=123.45,
+        coarse_aggregates=67.890,
+        fa_density='test FA density',
```

Comment:
fa_density an ca_density actually should be some number input. For the test it is not crucial, but it makes it more clear what actual data could be expected. I know that currently the two fields are modelled as string fields in WTForms. I already changed this to DecimalField on the branch I am currently working on. Further, note the comments above concerning number as strings vs. actual floats. We can discuss later.